source/features/latest-tag-button.tsx

```diff
@@ -95,7 +95,9 @@ async function init(): Promise<false | void> {
 	if (pageDetect.isRepoRoot()) {
 		href = `/${getRepoURL()}/tree/${latestTag}`;
 	} else {
-		href = replaceBranch(currentBranch, latestTag);
+		const pathParts = parseRoute(location.pathname);
+		pathParts[5] = latestTag;
```

Comment:
5?

Reply:
Branch Name

Reply:
@fregante good catch. Not sure how I managed that!